lib/UserManagement.php

```diff
@@ -278,6 +281,134 @@ public function listOrganizationMemberships(
         return [$before, $after, $organizationMemberships];
     }
 
+    /**
+     * Sends an Invitation
+     *
+     * @param string $email The email address of the invitee
+     * @param string $organizationId Organization ID
+     * @param int $expiresInDays expiration delay in days
+     * @param string $inviterUserId User ID of the inviter
+     * @throws Exception\WorkOSException
+     *
+     * @return \WorkOS\Resource\Invitation
```

Comment:
i noticed for some other endpoints we use
```
@return \WorkOS\Resource\Response
```

is one better or more accurate than the other?

Reply:
Response is before parsing, it's the raw http request. I want to return the parsed entities, which are all extending BaseWorkOSResource

Reply:
got it. so i think that means the other methods denoted with `@return \WorkOS\Resource\Response` are inaccurate? can you review them in a followup please 🙏

Reply:
I'll check the UserManagement.

Reply:
perfect thanks!